website/src/components/symbols/HeaderImageLeft.js

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+const HeaderImageLeft = ({ width, height }) => {
+	return (
+		<svg aria-labelledby="title-energy-left" width={width} height={height} viewBox="0 0 360 228">
```

Comment:
```suggestion
		<svg width={width} height={height} viewBox="0 0 360 228">
```

Reply:
The aria label and titles were in the existing svg files. Have updated now 👍